collect_app/src/test/java/org/odk/collect/android/views/TrackingTouchSliderTest.java

```diff
@@ -0,0 +1,45 @@
+package org.odk.collect.android.views;
+
+import android.widget.LinearLayout;
+
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.core.view.MotionEventBuilder;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.odk.collect.android.R;
+import org.robolectric.RobolectricTestRunner;
+
+import static android.view.MotionEvent.ACTION_DOWN;
+import static android.view.MotionEvent.ACTION_UP;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+@RunWith(RobolectricTestRunner.class)
+public class TrackingTouchSliderTest {
+
+    private TrackingTouchSlider slider;
+
+    @Before
+    public void setUp() {
+        ApplicationProvider.getApplicationContext().setTheme(R.style.Theme_Collect_Light);
+
+        LinearLayout linearLayout = new LinearLayout(ApplicationProvider.getApplicationContext());
+        slider = new TrackingTouchSlider(ApplicationProvider.getApplicationContext(), null);
+
+        linearLayout.addView(slider);
+    }
+
+    @Test
+    public void onStartTrackingTouch_suppressesFlingGesture() {
+        slider.onTouchEvent(MotionEventBuilder.newBuilder().setAction(ACTION_DOWN).build());
+        assertThat(slider.isTrackingTouch(), equalTo(true));
+    }
+
+    @Test
+    public void onStopTrackingTouch_doesNotSuppressFlingGesture() {
```

Comment:
Probably worth combining this with the test above so it tests that the `ACTION_UP` **after** the `ACTION_DOWN` works. With these tests the implementation could just set `isTrackingTouch` to `true` on an `ACTION_DOWN` and then never change it back 😉